[2024-08-03T08:30:49.504+0000] {processor.py:161} INFO - Started process (PID=1826) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:30:49.505+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:30:49.507+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:30:49.506+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:30:49.510+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:30:49.509+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:30:49.511+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:30:49.510+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:30:49.511+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:30:49.522+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-08-03T08:31:43.400+0000] {processor.py:161} INFO - Started process (PID=1887) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:31:43.401+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:31:43.403+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:31:43.402+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:31:43.406+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:31:43.406+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:31:43.407+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:31:43.407+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:31:43.407+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:31:43.418+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:33:17.311+0000] {processor.py:161} INFO - Started process (PID=1991) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:33:17.313+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:33:17.315+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:33:17.314+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:33:17.318+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:33:17.318+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:33:17.319+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:33:17.319+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:33:17.319+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:33:17.330+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:34:13.201+0000] {processor.py:161} INFO - Started process (PID=2048) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:34:13.203+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:34:13.205+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:34:13.204+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:34:13.208+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:34:13.207+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:34:13.208+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:34:13.208+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:34:13.209+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:34:13.219+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-08-03T08:35:27.417+0000] {processor.py:161} INFO - Started process (PID=2138) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:35:27.419+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:35:27.421+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:35:27.420+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:35:27.424+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:35:27.423+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:35:27.425+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:35:27.424+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:35:27.425+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:35:27.436+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:36:20.420+0000] {processor.py:161} INFO - Started process (PID=2192) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:36:20.421+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:36:20.423+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:36:20.423+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:36:20.426+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:36:20.426+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:36:20.427+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:36:20.427+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:36:20.427+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:36:20.438+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-08-03T08:37:18.628+0000] {processor.py:161} INFO - Started process (PID=2259) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:37:18.630+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:37:18.632+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:37:18.631+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:37:18.635+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:37:18.634+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:37:18.635+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:37:18.635+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:37:18.636+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:37:18.647+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-08-03T08:38:09.759+0000] {processor.py:161} INFO - Started process (PID=2343) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:38:09.761+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:38:09.762+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:38:09.762+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:38:09.766+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:38:09.765+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:38:09.766+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:38:09.766+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:38:09.767+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:38:09.777+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-08-03T08:39:07.305+0000] {processor.py:161} INFO - Started process (PID=2405) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:39:07.307+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:39:07.308+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:39:07.308+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:39:07.312+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:39:07.311+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:39:07.312+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:39:07.312+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:39:07.313+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:39:07.324+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-08-03T08:40:02.082+0000] {processor.py:161} INFO - Started process (PID=2465) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:40:02.083+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:40:02.085+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:40:02.085+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:40:02.089+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:40:02.088+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:40:02.089+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:40:02.089+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:40:02.089+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:40:02.101+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:40:52.963+0000] {processor.py:161} INFO - Started process (PID=2525) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:40:52.964+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:40:52.966+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:40:52.965+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:40:52.969+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:40:52.969+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:40:52.970+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:40:52.970+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:40:52.970+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:40:52.981+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:41:41.934+0000] {processor.py:161} INFO - Started process (PID=2579) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:41:41.935+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:41:41.937+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:41:41.936+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:41:41.941+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:41:41.940+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:41:41.941+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:41:41.941+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:41:41.941+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:41:41.952+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:42:48.091+0000] {processor.py:161} INFO - Started process (PID=2653) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:42:48.092+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:42:48.094+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:42:48.094+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:42:48.098+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:42:48.097+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:42:48.098+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:42:48.098+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:42:48.098+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:42:48.110+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:43:51.312+0000] {processor.py:161} INFO - Started process (PID=2771) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:43:51.313+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:43:51.315+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:43:51.315+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:43:51.319+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:43:51.318+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:43:51.319+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:43:51.319+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:43:51.319+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:43:51.331+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:44:55.634+0000] {processor.py:161} INFO - Started process (PID=2950) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:44:55.635+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:44:55.637+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:44:55.636+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:44:55.640+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:44:55.640+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:44:55.641+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:44:55.641+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:44:55.641+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:44:55.652+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:46:00.986+0000] {processor.py:161} INFO - Started process (PID=3122) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:46:00.987+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:46:00.989+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:46:00.989+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:46:00.993+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:46:00.992+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:46:00.993+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:46:00.993+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:46:00.994+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:46:01.005+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:47:05.212+0000] {processor.py:161} INFO - Started process (PID=3193) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:47:05.214+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:47:05.216+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:47:05.215+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:47:05.219+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:47:05.218+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:47:05.219+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:47:05.219+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:47:05.220+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:47:05.231+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:48:10.965+0000] {processor.py:161} INFO - Started process (PID=3734) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:48:10.967+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:48:10.969+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:48:10.968+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:48:10.972+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:48:10.972+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:48:10.973+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:48:10.972+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:48:10.973+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:48:10.984+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-08-03T08:49:15.026+0000] {processor.py:161} INFO - Started process (PID=3925) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:49:15.028+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:49:15.030+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:49:15.029+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:49:15.033+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:49:15.033+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:49:15.034+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:49:15.034+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:49:15.034+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:49:15.046+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-08-03T08:50:21.182+0000] {processor.py:161} INFO - Started process (PID=4438) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:50:21.184+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:50:21.186+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:50:21.186+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:50:21.191+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:50:21.190+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:50:21.191+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:50:21.191+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:50:21.192+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:50:21.204+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2024-08-03T08:51:11.653+0000] {processor.py:161} INFO - Started process (PID=4576) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:51:11.654+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:51:11.656+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:51:11.656+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:51:11.660+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:51:11.660+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:51:11.661+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:51:11.661+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:51:11.661+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:51:11.672+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T08:52:02.024+0000] {processor.py:161} INFO - Started process (PID=4709) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:52:02.026+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:52:02.028+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:52:02.028+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:52:02.032+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:52:02.032+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:52:02.033+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:52:02.033+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:52:02.033+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:52:02.044+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T08:53:05.599+0000] {processor.py:161} INFO - Started process (PID=4836) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:53:05.601+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:53:05.603+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:53:05.602+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:53:05.607+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:53:05.606+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:53:05.607+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:53:05.607+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:53:05.607+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:53:05.618+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T08:54:12.988+0000] {processor.py:161} INFO - Started process (PID=4914) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:54:12.989+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:54:12.992+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:54:12.991+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:54:12.995+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:54:12.995+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:54:12.996+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:54:12.996+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:54:12.996+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:54:13.007+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T08:55:12.046+0000] {processor.py:161} INFO - Started process (PID=4987) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:55:12.047+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:55:12.050+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:55:12.049+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:55:12.054+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:55:12.053+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:55:12.054+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:55:12.054+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:55:12.054+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:55:12.066+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T08:56:18.695+0000] {processor.py:161} INFO - Started process (PID=5056) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:56:18.697+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:56:18.699+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:56:18.699+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:56:18.703+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:56:18.702+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:56:18.703+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:56:18.703+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:56:18.704+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:56:18.715+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T08:57:15.049+0000] {processor.py:161} INFO - Started process (PID=5187) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:57:15.051+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:57:15.053+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:57:15.053+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:57:15.057+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:57:15.056+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:57:15.057+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:57:15.057+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:57:15.058+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:57:15.069+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T08:58:26.362+0000] {processor.py:161} INFO - Started process (PID=5385) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:58:26.363+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:58:26.493+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:58:26.493+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:58:26.496+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:58:26.496+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:58:26.497+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:58:26.496+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:58:26.497+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:58:26.504+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.158 seconds
[2024-08-03T08:59:17.773+0000] {processor.py:161} INFO - Started process (PID=5468) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:59:17.775+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T08:59:17.777+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:59:17.776+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:59:17.781+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:59:17.780+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T08:59:17.781+0000] {logging_mixin.py:188} INFO - [2024-08-03T08:59:17.781+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T08:59:17.781+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T08:59:17.792+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-08-03T09:00:06.178+0000] {processor.py:161} INFO - Started process (PID=5609) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:00:06.180+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:00:06.182+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:00:06.181+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:00:06.186+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:00:06.185+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:00:06.186+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:00:06.186+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:00:06.186+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:00:06.198+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2024-08-03T09:01:08.968+0000] {processor.py:161} INFO - Started process (PID=5919) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:01:08.969+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:01:08.971+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:01:08.971+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:01:08.975+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:01:08.975+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:01:08.976+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:01:08.976+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:01:08.976+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:01:08.988+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2024-08-03T09:02:13.726+0000] {processor.py:161} INFO - Started process (PID=6174) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:02:13.728+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:02:13.730+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:02:13.729+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:02:13.734+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:02:13.733+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:02:13.734+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:02:13.734+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:02:13.734+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:02:13.746+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2024-08-03T09:03:15.758+0000] {processor.py:161} INFO - Started process (PID=6364) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:03:15.760+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:03:15.762+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:03:15.762+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:03:15.766+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:03:15.766+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:03:15.767+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:03:15.766+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:03:15.767+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:03:15.778+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2024-08-03T09:04:16.116+0000] {processor.py:161} INFO - Started process (PID=6497) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:04:16.117+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:04:16.119+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:04:16.119+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:04:16.123+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:04:16.122+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:04:16.124+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:04:16.123+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:04:16.124+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:04:16.135+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2024-08-03T09:05:22.311+0000] {processor.py:161} INFO - Started process (PID=6599) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:05:22.313+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:05:22.315+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:05:22.315+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:05:22.319+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:05:22.318+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:05:22.319+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:05:22.319+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:05:22.320+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:05:22.331+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T09:06:28.576+0000] {processor.py:161} INFO - Started process (PID=6870) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:06:28.578+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:06:28.579+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:06:28.579+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:06:28.583+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:06:28.582+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:06:28.583+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:06:28.583+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:06:28.584+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:06:28.595+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T09:07:34.620+0000] {processor.py:161} INFO - Started process (PID=7145) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:07:34.621+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:07:34.623+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:07:34.623+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:07:34.627+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:07:34.626+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:07:34.628+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:07:34.627+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:07:34.628+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:07:34.639+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T09:08:39.151+0000] {processor.py:161} INFO - Started process (PID=7401) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:08:39.152+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:08:39.154+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:08:39.154+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:08:39.158+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:08:39.157+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:08:39.158+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:08:39.158+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:08:39.159+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:08:39.170+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T09:09:43.348+0000] {processor.py:161} INFO - Started process (PID=7737) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:09:43.350+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:09:43.352+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:09:43.351+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:09:43.355+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:09:43.355+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:09:43.356+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:09:43.356+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:09:43.356+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:09:43.367+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-08-03T09:10:45.823+0000] {processor.py:161} INFO - Started process (PID=7892) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:10:45.824+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:10:45.827+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:10:45.826+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:10:45.830+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:10:45.830+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:10:45.831+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:10:45.831+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:10:45.831+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:10:45.842+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T09:11:49.465+0000] {processor.py:161} INFO - Started process (PID=8032) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:11:49.467+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:11:49.469+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:11:49.468+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:11:49.473+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:11:49.472+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:11:49.473+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:11:49.473+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:11:49.473+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:11:49.484+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-03T09:12:49.247+0000] {processor.py:161} INFO - Started process (PID=8371) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:12:49.249+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-03T09:12:49.251+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:12:49.250+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:12:49.255+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:12:49.254+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-03T09:12:49.255+0000] {logging_mixin.py:188} INFO - [2024-08-03T09:12:49.255+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-03T09:12:49.255+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-03T09:12:49.266+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
